docs/_docs/Developers/intellij-idea.md

```diff
@@ -18,7 +18,7 @@ add: JavaFXApplication with:
 * main class: org.phoenicis.javafx.JavaFXApplication
 * VM options: 
 ```
---add-modules=ALL-MODULE-PATH --module-path=phoenicis-dist/target/lib
+--add-modules=ALL-MODULE-PATH --module-path=phoenicis-dist/target/lib -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI -classpath phoenicis-dist/target/lib --upgrade-module-path=phoenicis-dist/target/lib/compiler.jar
```

Comment:
I don't understand why -classpath would be required for GraalVM and not other dependencies. This might cause package problems

Reply:
Can you try if it works for you without it?